recipes/urbansim/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "urbansim" %}
+{% set version = "3.2" %}
+{% set sha256 = "c3ded3bd19b7ea7cedfa1ae50af9ccd9ec9da6dd9585adbf0b776e6c92b89e1e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```

Comment:
The upstream `setup.py` specifies a number of version restrictions on the dependencies:

https://github.com/UDST/urbansim/blob/dev/setup.py#L32-L42

Unless there is a good reason to ignore it, we should include them here.

Reply:
Added!

I didn't include the detail where `statsmodels` depends on the Python version -- this is a pip-specific issue that I haven't run into with conda. (Statsmodels v0.11 doesn't run in Python <3.6, but pip tries to install it anyway.)